.editorconfig

```diff
@@ -6,7 +6,7 @@
 charset = utf-8
 end_of_line = lf
 indent_size = 4
-indent_style = tab
+indent_style = space
```

Comment:
Rebuilt at https://github.com/silverstripe/silverstripe-cms/pull/1727. I'll revert from this PR.